iceoryx_posh/source/roudi/roudi_config_toml_file_provider.cpp

```diff
@@ -66,7 +66,20 @@ iox::cxx::expected<iox::RouDiConfig_t, iox::roudi::RouDiConfigFileParseError> To
     }
     auto groupOfCurrentProcess = iox::posix::PosixGroup::getGroupOfCurrentProcess().getName();
 
-    std::shared_ptr<cpptoml::table> parsedFile = cpptoml::parse_file(m_customConfigFilePath.c_str());
+    std::shared_ptr<cpptoml::table> parsedFile{nullptr};
```

Comment:
A `std::shared_ptr` is always initialized to `nullptr` so you do not need to do this explicitly. But for now, you can leave it as it is.
The initialization is only important for POD types like a pointer or int or something like this.

Reply:
Thanks for the hint!